Assets/Python/Screens/PlatyBuilder/WBPlotScreen.py

```diff
@@ -619,13 +623,13 @@ def handleInput(self, inputClass):
 		elif inputClass.getFunctionName().find("BaseYield") > -1:
 			i = YieldTypes(inputClass.getData2())
 			if inputClass.getData1() == 1030:
-				CyGame().setPlotExtraYield(pPlot.getX(), pPlot.getY(), i, iChange)
+				CyGame().setPlotExtraYield(pPlot.getX(), pPlot.getY(), i, abs(iChange))
```

Comment:
I changed the mechanics of the global variable iChange. Previously, iChange was just a positive integer. There was a different boolean which made that value positive or negative. Many screen had a dropdown menu which switches that boolean between True and False.

When you change something, like the yield production of a tile, it first looks at the boolean value to see if it should add or remove and then change it by the value iChange.
The example you highlighted is a case when there are 2 buttons to change the value. The + button should always increase the value and the - button should always decrease the value.

I changed it so that the add/remove dropdown menu doesn't change that specific boolean, but it switches the sign of the integer.
To make sure that the buttons still work properly, I had to make it absolute values. They make sure a + buttons always add and the - buttons always subtract, no matter if iChange is positive or negative.

The reason why I did this is because the adding and removing consequent in all WB screen files. I synced all files to this respect. And it makes some changing function simpler because it only has to change the value by iChange. (It will automatically increase or decrease depending on the mode you have selected)